test/evergreen.yml

```diff
@@ -277,7 +283,13 @@ functions:
           export "PATH=/cygdrive/c/Python39:/cygdrive/c/Python39/Scripts:$PATH"
           export "PYTHONPATH=$(pwd)/../lang/python/wiredtiger):$(cygpath -w $(pwd)/../lang/python)"
         fi
-        ${test_env_vars|} ${python_binary|python3} ../test/suite/run.py ${unit_test_args|-v 2} ${smp_command|} 2>&1
+        if [ ${is_cmake_build|false} = true ]; then
+          cd cmake_build
+          ${test_env_vars|} WT_BUILDDIR=$(pwd) ${python_binary|python3} ../test/suite/run.py ${unit_test_args|-v 2} ${smp_command|} 2>&1
```

Comment:
Can `WT_BUILDDIR=$(pwd)` be moved into `${test_env_vars}` in the build variant setting, so that to have a common test command line outside of cmake vs autoconf checking logic?

Reply:
Yep, that's a good idea! I've updated the config, moving `WT_BUILDDIR=$(pwd)` into `${test_env_vars}`.